frontend/src/app/components/wp-relations/wp-relations-hierarchy/wp-relations-hierarchy.service.ts

```diff
@@ -136,9 +136,14 @@ export class WorkPackageRelationsHierarchyService {
         lockVersion: childWorkPackage.lockVersion
       }).then(wp => {
         if (parentWorkPackage) {
-          this.wpCacheService.require(parentWorkPackage.id!, true);
+          this.wpCacheService.require(parentWorkPackage.id!, true).then((wp) => {
+            this.halEvents.push(wp, {
+            eventType: 'association',
+            relatedWorkPackage: null,
+            relationType:''
```

Comment:
Hi @bsatarnejad please take care of correct spacings. There is a space missing after the colon. Further, please use "child" as the relationType.